src/lib/slider/slider.spec.ts

```diff
@@ -106,21 +110,21 @@ describe('MdSlider', () => {
     });
 
     it('should update the track fill on click', () => {
-      expect(trackFillElement.style.flexBasis).toBe('0%');
+      expect(trackFillElement.style.transform).toBe('scaleX(0)');
```

Comment:
Just curious, why change from `flex-basis` to `transform` instead of changing the flex axis? Just the general preference to `transform`?